Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp

```diff
@@ -50,6 +50,7 @@ namespace ROS2
                         &ROS2ImuSensorComponent::m_filterSize,
                         "Filter Length",
                         "Filter Length, large value allows to reduce numeric noise")
+                    ->Attribute(AZ::Edit::Attributes::Min, 1)
```

Comment:
Is there also a max value we can attribute here as well?

Reply:
I guess it does not make much sense to have a very large moving window. Maybe 200 could be a good maximum?

Reply:
Agree, add 200 maximum samples it will be 2 seconds for 100 Hz.

Reply:
I have added a max value attribute at 200 and changed UIHandler to Slider for a better user experience